packages/react-components/react-storybook-addon/README.md

```diff
@@ -68,3 +68,34 @@ module.exports = {
    - > 💡 this will run `prestorybook` script that compiles addon implementation with all of its direct dependencies that live within monorepo, so it can be consumed by local storybook
 
 2. Every time you do any change to implementation, after you ran your local storybook you'll need to manually run `yarn workspace @fluentui/react-storybook-addon build` to reflect those changes
+
+## Parameter Configuration
+
+- Three custom optional parameters can be set to alter behavior of the addon
+  1. `dir` - determines whether to render story in `ltr` or `rtl` mode. Default is `undefined`.
+  2. `fluentTheme` - determines whether to render story theme in `web-light`, `web-dark`, `teams-high-contrast`, `teams-dark`, or `teams-light`. Setting this
+     parameter will disable ability to dynamically change the theme within story canvas or doc.
+  3. `isVrTest` - when set to `true`, this removes injected padding and background theme that's automatically applied from rendered story.
+
+```js
+import { TEAMS_HIGH_CONTRAST, WEB_DARK, WEB_LIGHT } from '@fluentui/react-storybook-addon';
```

Comment:
**suggestion:** hmm best API is no API or very limited API surface 🙌. 

we don't need these tokens for sake of setting fluent theme. 

We can instead leverage one or both of the following approaches:

1. export only type that consumer can use to get proper intelissense for setting parameters 
```ts
import type {Parameters} from '@fluentui/react-storybook-addon';

export const ButtonDarkMode = {
  render: Button,
  parameters: { fluentTheme: 'web-dark' } as Parameters
};
```

2. export function that will provide all the intellisense without need to do any manual work

```ts
import {parameters} from '@fluentui/react-storybook-addon';

export const ButtonDarkMode = {
  render: Button,
  // parameters function has all the proper TS annotations so we will get top DX
  parameters: parameters({ fluentTheme: 'web-dark' })
};
```

```ts
// this will be basically an identity function
export function parameters(options:FluentAddonParameters){ 
  // possibly process default values here 
  return options;
}
```

Reply:
These two options combined to provide TS typing via intellisense are definitely a much better solution than constant tokens! I've made the change to add a `parameters` function and have removed the constants in favor of exporting a `FluentParameters` type.